src/utils/supabase.tsx

```diff
@@ -0,0 +1,6 @@
+import { createClient } from "@supabase/supabase-js";
+
+const SUPABASE_URL = "https://wahxmjacrbklrveozznp.supabase.co";
```

Comment:
teoretycznie URL do bazy może być inny dla różnych środowisk więc to też mogłoby być w .env